testcases/nameTest/subpackages_test.py

```diff
@@ -242,6 +243,9 @@ def _getSubPackages(self):
             subpackages.update(self._get_debuginfo())
             subpackages.update({"src" + suffix})
 
+        subpackages.discard('accessibility-slowdebug')
+        subpackages.discard('accessibility-fastdebug')
+
```

Comment:
Accessibility is tricky one.  Afaik it is built for all releas-fastdebug/slowdebug builds
Right now it is availabl in all jdks8 in all platfomrs. This is going to change.
It will remain on jdk8 in rhel8 and f32 and lower
However will be removed form jdk8 in f33 and up. Actually I already commited that to jdk8rpms in rawhide.
Once remaining issues will be solved, the accessibility subpackage will be added to jdk11 in f33 andup

Reply:
so what are you sayin is that accessibility was supposed to be in the ojdk8 packages and in the future it will be removed from ojdk8, but added into ojdk11? also, what about latest? I would leave it as it is right now and change it again once we see that those changes in rpms were made.